index.html

```diff
@@ -4853,6 +4853,67 @@ <h3>Issuer</h3>
         </p>
       </section>
 
+      <section class="informative">
+        <h4>Holder</h4>
+        <p>
+The value associated with the <code>holder</code> <a>property</a> is expected
+to identify a <a>holder</a> that is known and trusted by the
```

Comment:
Another option that might be simpler:

```suggestion
The value associated with the <code>holder</code> <a>property</a> is expected
to identify the <a>holder</a> to the verifier.
```

Reply:
+1, that would resolve my concern.